identity-service/sequelize/sequelize-config.json

```diff
@@ -1,5 +1,11 @@
 {
   "development": {
+    "username": "postgres",
```

Comment:
This was for locally running migrations for running commands like`npx sequelize-cli db:migrate`.
I figured it might be useful for the future, but can remove.

Reply:
This theoretically shouldn’t be needed. The service init should run migrations before the web server comes up. It connects to Postgres using the dbUrl from the configs. Did it not run migrations for you?